fireplace/dsl/selector.py

```diff
@@ -371,6 +371,7 @@ def CONTROLLED_BY(selector):
 ALWAYS_WINS_BRAWLS = AttrValue(enums.ALWAYS_WINS_BRAWLS) == True
 KILLED_THIS_TURN = AttrValue(enums.KILLED_THIS_TURN) == True
 
+ROGUE = AttrValue(GameTag.CLASS) == 7
```

Comment:
The CLASS_CARD helper is a EnumSelector and does not implement the __call__ func, thus I cannot just call it